src/client/App.jsx

```diff
@@ -1,56 +1,119 @@
-import { useEffect, useState } from 'react';
-import './App.css';
-import MovieForm from './components/MovieForm';
-import UserForm from './components/UserForm';
+import { useEffect, useState } from "react";
+import { Routes, Route, Link, useNavigate } from "react-router-dom";
+import "./App.css";
+import MovieForm from "./components/MovieForm";
+import UserForm from "./components/UserForm";
 
-const apiUrl = 'http://localhost:4000';
+const apiUrl = "http://localhost:4000";
```

Comment:
Store these variables as env vars (use your .env file)

Reply:
Stored in .env file ✔️